### PR TITLE
OSDOCS-12394: GCP machineset single svc acct limit

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.12+

Issue:
[OSDOCS-12394](https://issues.redhat.com//browse/OSDOCS-12394)

Link to docs preview:
- [Sample YAML for a compute machine set custom resource on GCP](https://83906--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-gcp#machineset-yaml-gcp_creating-machineset-gcp) regular compute
- [Sample YAML for a compute machine set custom resource on GCP](https://83906--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets#machineset-yaml-gcp_creating-infrastructure-machinesets) infra
- [Sample GCP provider specification](https://83906--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-gcp#cpmso-yaml-provider-spec-gcp_cpmso-config-options-gcp) control plane

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by JoelSpeed — COMMENTED** [review]
LGTM

File: modules/cpmso-yaml-provider-spec-gcp.adoc [hunk 1]
```diff
@@ -93,5 +93,6 @@ To use a GCP Marketplace image, specify the offer to use:
 <3> Specifies the cloud provider platform type. Do not change this value.
 <4> Specifies the name of the GCP project that you use for your cluster.
 <5> Specifies the GCP region for the cluster.
-<6> Specifies the control plane user data secret. Do not change this value.
-<7> This parameter is configured in the failure domain, and is shown with an empty value here. If a value specified for this parameter differs from the value in the failure domain, the Operator overwrites it with the value in the failure domain.
\ No newline at end of file
+<6> Specifies a single service account. Multiple service accounts are not supported. 
+<7> Specifies the control plane user data secret. Do not change this value.
+<8> This parameter is configured in the failure domain, and is shown with an empty value here. If a value specified for this parameter differs from the value in the failure domain, the Operator overwrites it with the value in the failure domain.
```
Comment (agantony) [inline modules/cpmso-yaml-provider-spec-gcp.adoc:98]:
```suggestion
<8> This parameter is configured in the failure domain, and is shown with an empty value. If the specified value for this parameter differs from the one in the failure domain, the Operator overwrites it with the value from the failure domain.
```

**Review by agantony — COMMENTED** [review]
Added a suggestion for your consideration; overall lgtm!

**Pull request merged**